.travis.yml

```diff
@@ -9,13 +9,21 @@ addons:
       - ubuntu-toolchain-r-test
     packages:
       - g++-4.8
+      - python3-pip
 cache:
   directories:
     - node_modules
 before_script:
-  - pwd
-  - cp ~/build/HackIllinois/api/config/test.config.template ~/build/HackIllinois/api/config/test.config
   - npm install
+  - cp ./config/test.json.template ./config/test.json
+after_success:
```

Comment:
We can also get rid of this section as disabling cleanup should purge the test config